project/Versions.scala

```diff
@@ -2,7 +2,7 @@ package org.scalameta
 package build
 
 object Versions {
-  val LatestScala211 = "2.11.12"
+  val LatestScala211 = "2.13.2"
```

Comment:
@fthomas you may want to configure Scala Steward to ignore updates on `org.scala-lang:scala*` artifacts like in this PR

Reply:
Agreed. We're already ignoring [`scala-compiler`, `scala-library`, and `scala-reflect`](https://github.com/fthomas/scala-steward/blob/111e2f415e1a6cc9ea268e22d4daed751bdb315d/modules/core/src/main/scala/org/scalasteward/core/update/FilterAlg.scala#L78-L80) and I'll add `scalap` to the list.

Thanks for the notice, @olafurpg!